<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SchedulerDialogue</class>
 <widget class="QDialog" name="SchedulerDialogue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>347</width>
    <height>262</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="title">
      <string>Scheduler</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="StartTimeLB">
        <property name="text">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt; color:#ffffff;&quot;&gt;Start Time:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="3">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QComboBox" name="HoursSB"/>
        </item>
        <item>
         <widget class="QComboBox" name="MinsSB"/>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="ScheduledonLB">
        <property name="text">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt; color:#ffffff;&quot;&gt;Scheduled On:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="SaturdayCB">
        <property name="text">
         <string>Saturday</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QCheckBox" name="SundayCB">
        <property name="text">
         <string>Sunday</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QCheckBox" name="MondayCB">
        <property name="text">
         <string>Monday</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="TuesdayCB">
        <property name="text">
         <string>Tuesday</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QCheckBox" name="WednesdayCB">
        <property name="text">
         <string>Wednesday</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QCheckBox" name="ThursdayCB">
        <property name="text">
         <string>Thursday</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="FridayCB">
        <property name="text">
         <string>Friday</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SchedulerDialogue</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SchedulerDialogue</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SaturdayCB</sender>
   <signal>clicked(bool)</signal>
   <receiver>SchedulerDialogue</receiver>
   <slot>setOK(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>12</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>TuesdayCB</sender>
   <signal>clicked(bool)</signal>
   <receiver>SchedulerDialogue</receiver>
   <slot>setOK(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>17</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SundayCB</sender>
   <signal>clicked(bool)</signal>
   <receiver>SchedulerDialogue</receiver>
   <slot>setOK(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>11</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>WednesdayCB</sender>
   <signal>clicked(bool)</signal>
   <receiver>SchedulerDialogue</receiver>
   <slot>setOK(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>208</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>9</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MondayCB</sender>
   <signal>clicked(bool)</signal>
   <receiver>SchedulerDialogue</receiver>
   <slot>setOK(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>287</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>388</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ThursdayCB</sender>
   <signal>clicked(bool)</signal>
   <receiver>SchedulerDialogue</receiver>
   <slot>setOK(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>305</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>392</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>groupBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>SchedulerDialogue</receiver>
   <slot>setOK(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>118</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>391</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>FridayCB</sender>
   <signal>clicked(bool)</signal>
   <receiver>SchedulerDialogue</receiver>
   <slot>setOK(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>192</y>
    </hint>
    <hint type="destinationlabel">
     <x>7</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setOK(bool)</slot>
 </slots>
</ui>
